src/helpers/figures/charts/line_chart.ts

```diff
@@ -251,21 +257,25 @@ function getChartAxisType(chart: LineChart, getters: Getters): AxisType {
 }
 
 function isDateChart(chart: LineChart, getters: Getters): boolean {
-  return !chart.labelsAsText && canBeDateChart(chart.labelRange, getters);
+  return !chart.labelsAsText && canBeDateChart(chart.labelRange, chart.dataSetsHaveTitle, getters);
 }
 
 function isLinearChart(chart: LineChart, getters: Getters): boolean {
   return !chart.labelsAsText && canBeLinearChart(chart.labelRange, getters);
 }
 
-function canBeDateChart(labelRange: Range | undefined, getters: Getters): boolean {
+function canBeDateChart(
+  labelRange: Range | undefined,
+  dataSetsHaveTitle: boolean,
+  getters: Getters
+): boolean {
   if (!labelRange || !canBeLinearChart(labelRange, getters)) {
     return false;
   }
   const labelFormat = getters.getEvaluatedCell({
     sheetId: labelRange.sheetId,
     col: labelRange.zone.left,
-    row: labelRange.zone.top,
+    row: dataSetsHaveTitle ? labelRange.zone.top + 1 : labelRange.zone.top,
```

Comment:
Hello :wave: 

There's an alternative implementation that might be less invasive in all the code (additional parameter to a lot of functions) and even better from a functional point of view.

The idea would be to check *all* cells in the label range, one after the other (starting with the first empty one if "dataSetsHaveTitle") and return the first format found.

Reply:
Thanks for the suggestion! I’ve removed the `dataSetsHaveTitle` parameter and updated the code accordingly :)

Reply:
@LucasLefevre : What if we have different formats in the label range ? We still keep the first found ?

Reply:
Yes. It's not worse than the current behavior (only from the first cell) and it's easy